apps/x509.c

```diff
@@ -457,6 +457,8 @@ int x509_main(int argc, char **argv)
         case OPT_MD:
             if (!opt_md(opt_unknown(), &digest))
                 goto opthelp;
+        default:
```

Comment:
There should be a `break;` above here to avoid fall-thru-type warnings.

Reply:
don't spend much time on reviewing this, until we see if the icc compiler is buggy about the enum/switch warning.

Reply:
Fair enough. WRT fall-thru comments/warnings, the upstream code is somewhat inconsistent.